config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   # Adding nested resources routes for users and posts
   resources :users , only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show] do
+      post 'addlike', on: :member
+      delete 'deletelike', on: :member
+      resources :comments, only: [:new, :create]
+    end
   end 
 
   # Defines the root path route ("/")
```

Comment:
Nice job so far👍🏼. currently, you still haven't defined a root path, kindly add the root route to lead to the list of users😉. see the suggestion below👍🏼
<img width="1224" alt="image" src="https://github.com/NitBravoA92/blog-app/assets/64643995/112cc62e-2538-4838-9f43-8ed742179d75">


```suggestion
    resources :posts, only: [:index, :new, :create, :show] do
      post 'addlike', on: :member
      delete 'deletelike', on: :member
      resources :comments, only: [:new, :create]
    end
  end 

  # Defines the root path route ("/")
  
  root "users#index"

```